blocks/hooks/index.js

```diff
@@ -14,36 +14,22 @@ import matchers from './matchers';
 const hooks = createHooks();
```

Comment:
Will we drop block-specifics hooks registry in a future PR in favor of the new singleton instance?

Reply:
This would make it possible to update block related hooks using both `wp.blocks.addFilter` and `wp.hooks.addFilter`, which I wanted to avoid. Documentation promotes the former, so that's why we might keep it as it is. I'm open for discussion about this one. It seems not consistent to register block with `wp.blocks.registerBlockType`, but update it with `wp.hooks.addFilter`. Hope it helps to understand the decision process here.

Reply:
It looks like we are dropping block-specific hooks with #3827.